src/plugins/embeddable/kibana.json

```diff
@@ -4,6 +4,7 @@
   "server": false,
   "ui": true,
   "requiredPlugins": [
+    "data",
```

Comment:
Are we comfortable with this explicit dependency? 
Are there nicer options? 

I wonder if isValueClickTriggerContext / isRangeSelectTrigger context should better go to `data` plugin then? or `uiActions`?

Reply:
I think we are moving in direction of actually having everything as an explicit dependencies. Like even when a plugin just imports some static code from another plugin it is good to capture that dependency relationship in `kibana.json`. Or for example [these improvements to `ui_actions`](https://github.com/elastic/kibana/issues/62225) to make trigger dependencies explicit.

> Are we comfortable with this explicit dependency?
> Are there nicer options?

I'm comfortable with this for now as these helper functionions can only be in `embeddable` plugin now, without further larger refactoring.

> I wonder if isValueClickTriggerContext / isRangeSelectTrigger context should better go to `data` plugin then? or `uiActions`?

Good question. I probably cannot be in `data` plugin as we had race conditions and @mattkime moved those triggers out of the data plugin https://github.com/elastic/kibana/pull/60168 https://github.com/elastic/kibana/pull/60202

We technically could have them in `ui_actions`, but it seems weird that `ui_actions` should know anything about visualizations.

`embeddable` seems about the right place for them, or maybe `visualizations` plugin could house them.